autoscale/src/main/resources/schema/app/20221110093257_CB-19545_Add_new_entity_to_record_autoscaling_triggers.sql

```diff
@@ -16,7 +16,7 @@ create table if not exists scaling_activity (
 	constraint fk_scaling_activity_cluster_id foreign key (cluster_id) references cluster(id)
 );
 
-create unique index if not exists idx_activity_crn on scaling_activity(activity_crn);
+create unique index if not exists idx_operation_id on scaling_activity(operation_id);
```

Comment:
All the changes here need to be an 'alter table' - right?

These changes are already deployed, and the tables will already exist on the various mow environments.

Reply:
yes, existing clusters should be handled on mow envs